README.md

```diff
@@ -11,179 +10,188 @@ for applicable air flight routes from a previously populated database. An additi
+    npm install
+    ```
 
-You can then run the app using your preferred Node toolchain. As you are running the server
-on your localhost, and not in a Docker container, you will have to point at it with the
+Note that `nodemon` is installed as a dev-dependency, so you can run the server with
+the benefit of automatic restarting as you make changes.
+
+You will have to point your application at the Couchbase server with the
 `CB_HOST` environment variable.
 
 The first time you run against a new database image, you may want to use the provided
 `wait-for-couchbase.sh` wrapper to ensure that all indexes are created.
 For example, using the Docker image provided:
 
-```
-docker-compose -f mix-and-match.yml up db
-CB_HOST=localhost ./wait-for-couchbase.sh node index.js
 
-# or, if you know the database is already configured and responding
-CB_HOST=localhost node index.js
-```
+    ```
+    docker-compose -f mix-and-match.yml up db
 
-If you already have an existing Couchbase server running and correctly configured, you might run:
+    export CB_HOST=localhost
+    ./wait-for-couchbase.sh echo "Couchbase is ready!"
+    npx nodemon index.js
+    ```
 
-```
-CB_HOST=10.144.211.101 CB_USER=Administrator CB_PSWD=password node index.js
-```
+If you already have an existing Couchbase server running and correctly configured, you might run:
 
-It's often convenient to use `nodemon` to auto-restart the app whenever you make a change.
+    ```
+    CB_HOST=10.144.211.101 CB_USER=Administrator CB_PSWD=password npx nodemon index.js
+    ```
 
-```
-npm install -g nodemon
-CB_HOST=localhost nodemon index.js
-```
+Finally, if you want to see how the sample frontend Vue application works with your changes,
+run it with:
 
-To run the front-end Vue application:
+    ```
+    docker-compose -f mix-and-match.yml up frontend
+    ```
 
-```
-docker-compose -f mix-and-match.yml up frontend
-```
 
-### Running the database or front-end manually
+### Running the front-end manually
 
-To run the database and/or frontend components manually without Docker, follow the appropriate guides:
+To run the frontend components manually without Docker, follow the guide
+[here](https://github.com/couchbaselabs/try-cb-frontend-v2)
 
-* [Couchbase server - getting started](https://docs.couchbase.com/server/current/getting-started/start-here.html)
-* [Travel sample frontend](https://github.com/couchbaselabs/try-cb-frontend-v2)
 
-## Configuration Options
+## REST API reference
 
-By default the application will use the `travel-sample.inventory` scope to query/search flight and hotel data.
+We've integrated Swagger/OpenApi version 3 documentation which can be accessed on the backend at `http://localhost:8080/apidocs`
 
-It will also separate user account data such as flight bookings and credentials in the `tenant_agent_XX scopes`.
+More detail on the design of the API can be found  at [https://github.com/couchbaselabs/try-cb-frontend/blob/master/documentation/try-cb-api-spec-v2.adoc]
```

Comment:
I don't think we should point to this as it is going to be old or deprecated. The swagger API should probably be enough.

Reply:
Good call! Are there details in the doc that should be added to the Swagger?
e.g. unlike most API docs, which are purely reference, as this is an example, it's useful to give some description of what's happening behind the scenes like the doc does.
(though not actual code samples like that has)

Reply:
Yeah we could add more to the descriptions in swagger for parameters etc, like so: https://github.com/couchbaselabs/try-cb-frontend/blob/master/documentation/try-cb-api-spec-v2.adoc#211-parameters.

Not sure if we need to show the queries, as you can see them in the code anyways, and it could change in future...

I do wonder if more information on the application might be more suited here though: https://docs.couchbase.com/nodejs-sdk/current/hello-world/sample-application.html